docs/fundamentals/flow/flow-yaml.md

```diff
@@ -0,0 +1,115 @@
+# Configure a Flow from YAML
+
+Instead of constructing a Flow in Python source codes, using a YAML file is the recommended way to configure Flows 
+because it is 
+
+- independent of the Python source codes,
+- easy to edit, maintain and extend,
+- human-readable.
+
+## Load and Save a Flow configuration
+`load_config()` is used to load the Flow configuration from a YAML file. Correspondingly, one uses `save_config()`  to 
```

Comment:
may we add this to the Flow API?

Reply:
Do you mean this section or the whole chapter? One reason why I didn't put the whole chapter into the Flow API part is because there are some parts about creating Flows.